app/src/main/java/com/geekskool/dhobi/Util.java

```diff
@@ -0,0 +1,11 @@
+package com.geekskool.dhobi;
+
+/**
+ * Created by manisharana on 22/2/17.
+ */
+public class Util {
+    public static boolean isInvalid(String name) {
```

Comment:
Actually, prefer including both negative and positive:

isValid
isNotValid -- calls `!isValid()` under the covers